src/components/AudioDetails/Related.vue

```diff
@@ -3,22 +3,59 @@
     <h4 class="b-header mb-6">
       {{ $t('audio-details.related-audios') }}
     </h4>
-    <AudioTrack
-      v-for="audio in relatedAudios"
-      :key="audio.id"
-      :audio="audio"
-      :is-compact="true"
-      layout="row"
-      class="mb-12"
-    />
+    <template v-if="!$fetchState.error">
+      <AudioTrack
+        v-for="audio in audios"
+        :key="audio.id"
+        :audio="audio"
+        layout="row"
+        size="m"
+        class="mb-12"
+      />
+      <LoadingIcon v-show="$fetchState.pending" />
+    </template>
+    <p v-show="!!$fetchState.error">{{ errorMessage }}</p>
   </aside>
 </template>
 
 <script>
+import { ref } from '@nuxtjs/composition-api'
+import { AUDIO } from '~/constants/media'
+import useRelated from '~/composables/use-related'
+
 export default {
   name: 'RelatedAudios',
   props: {
-    relatedAudios: {},
+    audioId: {
+      type: String,
+      required: true,
+    },
+    service: {},
+  },
+  /**
+   * Fetches related audios on `audioId` change
+   * @param props
```

Comment:
Generally I think we'd still give `props` a type of `object` (at least that's what we did in Gutenberg, not sure if it's actually necessary):

```suggestion
   * @param {object} props
```